README.md

```diff
@@ -835,8 +835,205 @@ When everything is working as expected locally using `mp-test`, we can run
+  To ensure Internal ID is displayed ensure the Internal ID box is enabled by clicking in the bottom left icon and then the checkbox.
+
+  ![highlight of the checkbox where internal id can be enbaled](doc/enable_internal_id_column.png)
+
+  ![highlight of the internal id](doc/higlight_internal_id_column.png)
+  </details>
+
+  <details>
+    <summary><h3> What are the different log files?</h3></summary>
+
+  There are five types of log files: adapter, server, build, test, and validation logs. Each log file is prepended with the type of
+  log file followed by a number that represents rollover.
+
+  - server.log:
+  Contains all logs related to the HTTP sever inside the container. Server logs can't be modified since the server code comes packaged
+  inside the [base-adapter image](https://projects.registry.vmware.com/harbor/projects/46752/repositories/base-adapter/artifacts-tab) Python image.
+
+  - adapter.log
+  Contains all logs related to the adapter. Adapter logs are all the logs generated by adapter code (e.g., the test() method or the collect() methods inside
+  `app/adapter.py`).
+
+  - test.log
+  Contains all logs related to `mp-test`
+
+  - build.log
+  Contains all logs related to `mp-build`
+
+  - validation.log
+  Contains a log of the validations performed by mp-test on the collection results. This logs are only generated locally.
+</details>
+
+<details>
+  <summary><h3> How do I add logs to my adapter?</h3></summary>
+
+  The template adapter defines a logger variable which configures all logging for the adapter using [adapter_logging](https://github.com/vmware/vmware-aria-operations-integration-sdk/blob/main/lib/python/src/aria/ops/adapter_logging.py) from the python SDK.
+  To use the logger in any other files, import the python [logging](https://docs.python.org/3/library/logging.html) module. eg.
+
+  ```python3
+  import logging
+
+  logger = logging.getLogger(__name__)
+
+  def my_method():
+    logger.info("info log")
+    logger.warning("warning log")
+    logger.error("error log")
+    logger.debug("debug log")
+     ...
+  ```
+</details>
+
+
+<details>
+  <summary><h3> How do I change the log level (Server and Adapter)?</h3></summary>
+
+  Server and Adapter log levels are set inside the `loglevels.cfg`; this file is located in `logs/loglevels.cfg` locally and `$ALIVE_BASE/user/log/adapters/<ADAPTERKEY>_adapter3/<ADAPTER_INTERNAL_INSTANCE_ID>/loglevels.cfg` in the cloud proxy.
+  If the file does not exist, it will be generated after a collection/test collection.
+
+  ADAPTERKEY should match the name of the adapter used in the manifest.txt, and the ADAPTER_INTERNAL_INSTANCE_ID should match the Internal ID
+  found in VMware Aria Operations at Environment>Inventory>Adapter Instances>My Adapter Adapter Instance>Instance** in the rightmost column.
+  To ensure Internal ID is displayed ensure the Internal ID box is enabled by clicking in the bottom left icon and then the checkbox.
```

Comment:
This part is a little hard to read when rendered in markdown.

Some things that might help:
* put the all-caps variables backpacks
* Bold the navigation hierarchy stand out a bit more (maybe you meant to do that, I see `**` on one side), and separate the levels better (maybe use an actual arrow (` &rarr; `) instead of just `>`?)

Reply:
This looks much better. Maybe so something to set apart **My Adapter Adapter Instance** since the others are all literal, but that is a placeholder value?